src/Sylius/Bundle/TaxonomyBundle/Resources/translations/validators.en.yml

```
@@ -21,3 +21,4 @@ sylius:
         locale:
             not_blank: Please enter the locale.
             invalid: This value is not a valid locale.
+            unique: Translation in {{ value }} locale already exists.
```

just wondering if we should be more specific about it like 
```suggestion
            unique: Translation in {{ value }} locale code already exists.
```